pkg/files/s3_integration_test.go

```diff
@@ -0,0 +1,131 @@
+// Vikunja is a to-do list application to facilitate your life.
```

Comment:
Please add some end to end tests here for storing and retrieving files from s3 and locally, depending on the config. It should be possible to add minio to the CI, similar to how we're testing the ldap integration.

Reply:
<img width="1059" height="795" alt="image" src="https://github.com/user-attachments/assets/a0024c25-3e95-4ad8-99fb-a900cc854528" />

Compared with minio/minio, bitnami/minio adds the ability to create a default bucket. The test uses bitnamilegacy/minio because both Bitnami and MinIO recently stopped maintaining their docker images, and bitnami/minio was archived to bitnamilegacy/minio.
In future, we may need to switch to a more actively maintained S3-compatible project, but for now, we only use basic read/write functionality, so it‘s not a big issue.